docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: '3.8'
+
+services:
+  ffc-pay-file-sender-azurite:
+    image: mcr.microsoft.com/azure-storage/azurite
```

Comment:
`image: mcr.microsoft.com/azure-storage/azurite:3.16.0`

image it was pulling down without the tag version

is this something we should be doing within our services to ensure `latest` versions of these dependencies do not cause harm?

Reply:
We probably should be yes.  Got to start somewhere, therefore updated.